test/metabase/api/downloads_exports_test.clj

```diff
@@ -235,17 +238,22 @@
 
 (defn all-outputs!
   [card-or-dashcard opts]
-  (merge
-   (when-not (= (t2/model card-or-dashcard) :model/DashboardCard)
-     {:unsaved-card-download    (unsaved-card-download card-or-dashcard opts)
-      :public-question-download (public-question-download card-or-dashcard opts)
-      :card-download            (card-download card-or-dashcard opts)
-      :alert-attachment         (alert-attachment! card-or-dashcard opts)})
-   {:dashcard-download        (card-download card-or-dashcard opts)
-    :public-dashcard-download (public-dashcard-download card-or-dashcard opts)
-    :subscription-attachment  (subscription-attachment! card-or-dashcard opts)}))
+  (cond-> (merge
+           (when-not (= (t2/model card-or-dashcard) :model/DashboardCard)
+             {:unsaved-card-download    (delay (unsaved-card-download card-or-dashcard opts))
+              :public-question-download (delay (public-question-download card-or-dashcard opts))
+              :card-download            (delay (card-download card-or-dashcard opts))
+              :alert-attachment         (delay (alert-attachment! card-or-dashcard opts))})
+           {:dashcard-download        (delay (card-download card-or-dashcard opts))
+            :public-dashcard-download (delay (public-dashcard-download card-or-dashcard opts))
+            :subscription-attachment  (delay (subscription-attachment! card-or-dashcard opts))})
+    (:except opts) (#(apply dissoc % (:except opts)))
 
-(set! *warn-on-reflection* true)
+    ;; format rows and pivot does not support alert, they're all off by default
```

Comment:
UI doesn't let users configure pivot or format result option for alert, that's a dashboard subscription feature.